File: src/connect-to-db.R

# connect to postgresql to get our data

conn <- dbConnect(drv = PostgreSQL(), 
                  dbname = "sdad", 
                  host = "10.250.124.195", 
                  port = 5432, 
                  user = Sys.getenv("db_userid"), 
                  password = Sys.getenv("db_pwd"))

# query the users_gh data (table of all github users) 
db_data <- dbGetQuery(conn, "SELECT * FROM schema_name.table_name")

# disconnect from postgresql database 
dbDisconnect(conn)

File: src/connect-to-db.py


# how to get pandas data from postgree sql using python 
import psycopg2 as pg
import pandas.io.sql as psql
import pandas as pd
connection = pg.connect(host = 'postgis1', database = 'sdad', 
                        user = 'login', password = 'pwd')

postgreSQL_select_Query = '''
SELECT *  
FROM schema_name.table_name 
LIMIT 100
'''

df = pd.read_sql_query(postgreSQL_select_Query, con=connection)
df 
